### PR TITLE
HOTT-954: Added details section on RoO tab

### PR DESCRIPTION
**Pull request by jebw**
### Jira link

[HOTT-954](https://transformuk.atlassian.net/browse/HOTT-954)

### What?

I have added/removed/altered:

- [x] Added in the missing details section explaining how to use the rules of origin

### Why?

I am doing this because:

- This section of the page needs to be added

### CODE REVIEW

File: app/views/measures/_rules_of_origin_instructions.html.erb [hunk 1]
```diff
@@ -0,0 +1,598 @@
...
+        the rules (column 2)
+      </li>
+
+      <li>
+        a description of how non-originating materials must be worked or
+        processed to get country of origin status (column 3)
+      </li>
+    </ul>
+
+    <p>For each row:</p>
```
Comment (willfish) [inline app/views/measures/_rules_of_origin_instructions.html.erb:26]: Is this pseudo code or something the user should see?

File: app/views/measures/_rules_of_origin_instructions.html.erb [hunk 2]
```diff
@@ -0,0 +1,598 @@
...
+    <ul class="govuk-list--bullet">
+      <li>
+        check column 1 to find the product classification heading or chapter
+        number for the product being exported
+      </li>
+
+      <li>check column 2 to find the description that applies to the product</li>
+
+      <li>
+        check column 3 <!--(and 4 if present)//--> to see how the product must
```
Comment (willfish) [inline app/views/measures/_rules_of_origin_instructions.html.erb:37]: Should these html comments be visible to the user?

**Review by willfish — APPROVED** [review]
Seems fine :)

File: app/views/measures/_rules_of_origin_instructions.html.erb (hunk 2 above)
Comment (jebw) [reply app/views/measures/_rules_of_origin_instructions.html.erb:37]: Copy from Matt, I think their intentionally excluded

Comment (willfish) [reply app/views/measures/_rules_of_origin_instructions.html.erb:37]: Okey poke

**Pull request merged**